client/http/http.go

```diff
@@ -64,11 +97,16 @@ func (tc *TracedClient) Do(ctx context.Context, req *http.Request) (*http.Respon
 
 	req.Header.Set(correlation.HeaderID, correlation.IDFromContext(ctx))
 
+	start := time.Now()
 	rsp, err := tc.do(req)
 	if err != nil {
 		ext.Error.Set(ht.Span(), true)
 	} else {
 		ext.HTTPStatusCode.Set(ht.Span(), uint16(rsp.StatusCode))
+		if tc.metricsEnabled {
+			reqTotalMetric.WithLabelValues(req.Method, req.URL.Path, strconv.Itoa(rsp.StatusCode)).Inc()
```

Comment:
2 things to mention here:
- if the request error we do not capture any metric at all.
- We cannot use the path because the path might contain id's which will increase the cardinality of the metric.

Reply:
> We cannot use the path because the path might contain id's which will increase the cardinality of the metric.

Do you think we should use `Host` instead? And maybe introduce another vector dimension for the path?

Reply:
This may be more complicated than originally anticipated. Looking into it.

Reply:
Using `req.URL.Host` instead, as discussed.